libs/books/feature/src/lib/book-search/book-grid/book-grid-item/book-grid-item.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedTestingModule } from '@tmo/shared/testing';
+import { BooksFeatureModule } from '../../../books-feature.module';
+import { BookGridItemComponent } from './book-grid-item.component';
+
+describe('BookGridItemComponent', () => {
+  let component: BookGridItemComponent;
+  let fixture: ComponentFixture<BookGridItemComponent>;
+  const book = {
+      id: `${Date.now()}b`,
+      title: `Some Title ${Math.floor(Math.random() * 9999)}b`,
```

Comment:
There's no advantage to using a random title here. Why not just pick something readable?

Similarly with the dates and the id, there is no reason for them to be random. Just pick values.

Reply:
I used random values so we are not always testing against the same values, updating the tests to use hardcoded values now

Reply:
Tests should be, where possible, pure.